lib/_emerge/depgraph.py

```diff
@@ -3726,6 +3726,18 @@ def _eliminate_rebuilds(self):
                     if pkg.requires != installed_instance.requires:
                         continue
 
+                try:
+                    libc_deps = self._frozen_config._libc_deps_cache[pkg.root]
+                except (AttributeError, KeyError) as e:
+                    if isinstance(e, AttributeError):
+                        self._frozen_config._libc_deps_cache = {}
+
+                    self._frozen_config._libc_deps_cache[pkg.root] = find_libc_deps(
+                        self._frozen_config._trees_orig[pkg.root]["vartree"].dbapi,
+                        False,
+                    )
+                libc_deps = self._frozen_config._libc_deps_cache[pkg.root]
+
```

Comment:
We can add a libc_deps method, so as not to duplicate this code from _changed_deps.

Reply:
Added _installed_libc_deps method.